app/assets/stylesheets/provider/themes/wizard.scss

```diff
@@ -1,12 +1,31 @@
 // library
+
+html, body, header, div, span, main,
+h1, p, a, code, strong, ul, ol, li,
+form, fieldset, input, button, label {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+html {
+  line-height: 1;
+}
+
+ol, ul {
+  list-style: none;
+}
+
+header, main {
```

Comment:
https://github.com/Compass/compass/blob/1.0.3/core/stylesheets/compass/reset/_utilities.scss#L34C12-L34C23

html5 elements are reduced to only those used in the templates